.github/workflows/build-azure-uber-jar.yml

```
@@ -39,6 +39,32 @@ jobs:
           distribution: 'temurin'
           cache: 'maven'
 
+      - name: maven-settings-xml-action
+        uses: whelk-io/maven-settings-xml-action@v20
+        with:
+          repositories: |
+            [
+              {
+                "id": "file-replacer-maven-plugin",
+                "url": "https://maven.pkg.github.com/StevenMassaro/file-replacer-maven-plugin",
+                "releases": {
+                  "updatePolicy": "always",
+                  "enabled": "false"
+                },
+                "snapshots": {
+                  "enabled": "true",
+                  "updatePolicy": "always"
+                }
+              }
+            ]
```

This should be associated with [plugin_repositories](https://github.com/whelk-io/maven-settings-xml-action#plugin_repositories), not repositories. Also, `releases` should be `"enabled": "true"`

```suggestion
          plugin_repositories: |
            [
              {
                "id": "file-replacer-maven-plugin",
                "url": "https://maven.pkg.github.com/StevenMassaro/file-replacer-maven-plugin",
                "releases": {
                  "updatePolicy": "always",
                  "enabled": "true"
                },
                "snapshots": {
                  "enabled": "true",
                  "updatePolicy": "always"
                }
              }
            ]
```